src/components/Home/Home.js

```diff
@@ -0,0 +1,102 @@
+import React ,{ useEffect, useState } from 'react';
+import Button from '@material-ui/core/Button';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import { makeStyles } from '@material-ui/core/styles';
+import Container from '@material-ui/core/Container';
+import img from '../images/nike.jpg'
+
+import Divider from '@material-ui/core/Divider';
+import  Tab  from './Tab';
+import Item from '../Item/Item'
+import './Home.scss'
+
+import * as ApiService from '../ApiService/ApiService'
+import CollectionCard from './CollectionCard';
+
+const useStyles = makeStyles((theme) => ({
```

Comment:
again, remove this useStyles from js files and have separate .scss files for styles.